tests/src/test/scala/org/http4s/EntityDecoderSpec.scala

```diff
@@ -168,7 +168,7 @@ class EntityDecoderSpec extends Http4sSpec with PendingUntilFixed {
       "if decode is called with strict, will produce a MediaTypeMissing or MediaTypeMismatch " +
         "with ALL supported media types of the composite decoder" in {
         val reqMediaType          = MediaType.`text/x-h`
-        val expectedMediaRanges   = decoder1.consumes ++ decoder2.consumes ++ failDecoder.consumes
+        val expectedMediaRanges   = failDecoder.consumes ++ decoder1.consumes ++ decoder2.consumes
```

Comment:
I'm not sure why the order changed.  It's a questionable test because it depends on Set ordering.  But let's not let this hold up progress as long as it's stable.